erpnext_com/www/docs/user/manual/en/manufacturing/capacity-planning.md

```diff
@@ -0,0 +1,53 @@
+<!-- add-breadcrumbs -->
+# Capacity Planning
+
+Capacity planning is the process where an organization decides whether to accept the new orders or not based upon the resources and the existing work.
+
+Capacity planning has been enabled by default in your account, to know more go to:
+
+> Home > Manufacturing > Settings > Manufacturing Settings
+
+<img class="screenshot" alt="Work Order" src="{{docs_base_url}}/assets/img/manufacturing/capacity_planning_settings.png">
+
+## 1. Prerequisites
+Before creating and using a Work Order, it is advised that you create the following first:
+
+* [Bill Of Materials](/docs/user/manual/en/manufacturing/bill-of-materials)
+* [Operation](/docs/user/manual/en/manufacturing/operation)
+* [Workstation](/docs/user/manual/en/manufacturing/workstation)
+* [Work Order](/docs/user/manual/en/manufacturing/work-order)
+
+## 2. How Capacity Planning Works in ERPNext
+The user has to define the number of days in the "Capacity Planning For" under manufacturing settings to plan the upcoming work orders. For example, if you have kept the Capacity Planning For 30 days and to make 1 finished good required 5 days then on the current date user can only accept the 6 work orders. You can take the next work order when your work station gets free.
+
+### 2.1 Create Work Order With Operations
+User need to create the work orders with the operation so that the system will track the job card timings against the work order.
```

Comment:
```suggestion
User needs to create the Work Orders with Operations so that the system will track the [Job Card](/docs/user/manual/en/manufacturing/job-card) timings against the Work Order.
```